main.go

```diff
@@ -16,6 +16,8 @@ func main() {
 	cfg := pureport.NewConfiguration()
 
 	logCfg := ppLog.NewLogConfig()
+	logCfg.Level = "debug"
```

Comment:
Is this temporary? Will it leak information?

Reply:
This is only for the command line tool that which we will eventually support but for the moment is just for validation. In debug mode, we mask out API Secrets to show only the 4 four characters so that shouldn't be an issue.